test/jdk/java/nio/file/attribute/BasicFileAttributeView/libCreationTimeHelper.c

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2024 Alibaba Group Holding Limited. All Rights Reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+#include "export.h"
+#if (__STDC_VERSION__ >= 199901L)
+  #include <stdbool.h>
+#else
+  #define bool int
+  #define true 1
+  #define false 0
+#endif
```

Comment:
Do we really need this? Since https://openjdk.org/jeps/347 test libraries should be compiled with `--std=c11`.

Reply:
I am not quite sure, I copied these codes from `src/java.base/unix/native/libjsig/jsig.c`. I will check it.

Reply:
I think I need some time to study that.

Reply:
I found other tests such as `test/jdk/java/foreign/libIntrinsics.c` only `#include <stdbool.h>` to use bool type. So I think these codes can change to only `#include <stdbool.h>`.